unime/src-tauri/Cargo.toml

```diff
@@ -29,11 +29,16 @@ rustls.workspace = true
 rustls-platform-verifier.workspace = true
 serde_json.workspace = true
 tauri-plugin-barcode-scanner = { version = "2" }
+tauri-plugin-deep-link = { git = "https://github.com/tauri-apps/plugins-workspace", branch = "v2" }
```

Comment:
Why not just `{ version = "2" }` like the other declarations?

Reply:
because the update we needed was not published to crates.io yet when I made this, but it looks like they have now so I'll update it :)